rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/assembler/BulkOperationResultsRepresentationModelAssemblerTest.java

```diff
@@ -0,0 +1,63 @@
+package pro.taskana.task.rest.assembler;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import pro.taskana.common.api.BulkOperationResults;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.exceptions.ErrorCode;
+import pro.taskana.common.api.exceptions.TaskanaException;
+import pro.taskana.common.internal.util.EnumUtil;
+import pro.taskana.rest.test.TaskanaSpringBootTest;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.TaskState;
+import pro.taskana.task.api.exceptions.InvalidTaskStateException;
+import pro.taskana.task.rest.models.BulkOperationResultsRepresentationModel;
+
+@TaskanaSpringBootTest
+class BulkOperationResultsRepresentationModelAssemblerTest {
+
+  TaskanaEngine taskanaEngine;
+  TaskService taskService;
+  BulkOperationResultsRepresentationModelAssembler assembler;
+
+  @Autowired
+  BulkOperationResultsRepresentationModelAssemblerTest(
+      TaskanaEngine taskanaEngine,
+      TaskService taskService,
+      BulkOperationResultsRepresentationModelAssembler assembler) {
+    this.taskanaEngine = taskanaEngine;
+    this.taskService = taskService;
+    this.assembler = assembler;
+  }
+
+  @Test
+  void should_ReturnRepresentationModel_When_ConvertingEntityToRepresentationModel() {
+
+    BulkOperationResults<String, TaskanaException> result = new BulkOperationResults<>();
+    String taskId = "TKI:000000000000000000000000000000000003";
+    InvalidTaskStateException taskanaException =
+        new InvalidTaskStateException(
+            taskId, TaskState.COMPLETED, EnumUtil.allValuesExceptFor(TaskState.END_STATES));
+
+    result.addError("TKI:000000000000000000000000000000000003", taskanaException);
```

Comment:
Please use the variable "taskId" here instead of repeating the string.

Reply:
Ok I have changed it